src/main/java/uk/gov/pay/products/persistence/entity/PaymentEntity.java

```diff
@@ -106,7 +106,8 @@ public Payment toPayment() {
                 this.getAmount(),
                 this.product != null ? this.product.getExternalId() : null,
                 this.status,
-                this.product != null ? this.product.getId() : null
+                this.product != null ? this.product.getId() : null,
+                null
```

Comment:
Perhaps remove it from the constructor above, so that you don't need to pass null like this

Reply:
👍